portfolio/src/main/webapp/script.js

```diff
@@ -15,14 +15,22 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+ $(document).ready(function(){
+	$('.header').height($(window).height());
+})
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+function funFact() {
+  const quotes =
```

Comment:
For consistency, would it make sense to name this facts instead of quotes?

Reply:
Ok